datajunction-clients/python/datajunction/builder.py

```diff
@@ -494,7 +508,7 @@ def create_cube(  # pylint: disable=too-many-arguments
                 "tags": tags,
             },
             update_if_exists=update_if_exists,
-        )
+        )  # type: ignore
```

Comment:
Similar here, can be removed

Reply:
Here it is needed because Cube is not matching the `create_node` function return type of any node.